ecosystem/sep-0010.md

```diff
@@ -82,7 +82,8 @@ Request Parameters:
 Name      | Type          | Description
 ----------|---------------|------------
 `account` | `G...` string | The stellar account that the wallet wishes to authenticate with the server
-`home_domain` | string | (optional) The [fully qualified domain name](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of the service requiring authentication. SEP-10 signing servers used to generate tokens for multiple different web services can use this parameter to create valid tokens for each service. If not provided by the client, a default should be used.
+`home_domain` | string | (optional) The [fully qualified domain name](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of the service requiring authentication. SEP-10 signing servers used to generate tokens for multiple different web services can use this parameter to create valid tokens for each service. If not provided by the client, a default should be used
+`client_domain` | string | (optional) The [fully qualified domain name](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of the service that hosts the client organizations `stellar.toml` file. Only used for [Verifying Client Application Identity](#verifying-client-application-identity)
```

Comment:
We should state here or somewhere that if the client_domain parameter is provided, but not supported by the server, the server should ignore the value but continue as per normal. It's important clients can start including the field without servers returning an error.